config/FindVIGRANUMPY_DEPENDENCIES.cmake

```diff
@@ -92,9 +92,12 @@ IF(PYTHONINTERP_FOUND)
     # boost_python for the wrong Python version
     IF(Boost_FOUND)
         FIND_LIBRARY(Boost_PYTHON_LIBRARY
-                     NAMES boost_python-py${PYTHON_VERSION_MAJOR}${PYTHON_VERSION_MINOR}   # Linux with multiple Python versions
-                           boost_python-mt                                                 # Mac
-                           boost_python                                                    # default
+                     NAMES boost_python-py${PYTHON_VERSION_MAJOR}${PYTHON_VERSION_MINOR}      # Linux with multiple Python versions
+                           boost_python-py${PYTHON_VERSION_MAJOR}${PYTHON_VERSION_MINOR}-mt   # Linux with multiple Python versions (multithreaded)
+                           boost_python${PYTHON_VERSION_MAJOR}                                # Mac with Python 3
+                           boost_python${PYTHON_VERSION_MAJOR}-mt                             # Mac with Python 3 (multithreaded)
+                           boost_python                                                       # default
+                           boost_python-mt                                                    # Mac with Python (multithreaded)
```

Comment:
Perhaps. I am not sure I really follow what is getting multithreaded with the Python bindings. Do you know what is going on here?

Well, now that I think about it, I think we should be respecting `Boost_USE_MULTITHREADED`. ( https://cmake.org/cmake/help/v3.0/module/FindBoost.html ) That way we add the libraries by default, but if it is turned off we don't include them.